modules/localgov_services_landing/localgov_services_landing.module

```diff
@@ -25,6 +25,14 @@ function localgov_services_landing_preprocess_node(&$variables) {
       $variables['field_email_address'] = $node->get('field_email_address')->isEmpty() ? '' : $node->get('field_email_address')->first()->getValue()['value'];
       $variables['field_address_first_line'] = $node->get('field_address_first_line')->isEmpty() ? '' : $node->get('field_address_first_line')->first()->getValue()['value'];
       $variables['field_link_to_map'] = $node->get('field_link_to_map')->isEmpty() ? '' : $node->get('field_link_to_map')->first()->getValue()['uri'];
+
```

Comment:
The commented out code will need to be rewritten as part of the displaying statuses on a landing page. I was planning on handling that with the theming of service pages.

Reply:
Probably remove it and pop it in the appropriate follow-up issue.

Reply:
The whole hook function needs reviewing as part of the templating of the landing page. I was hoping to have done this in this sprint, but it will need to be done in the next sprint now. Until then I suggest we either remove the whole hook function or leave as is. 

I have created an issue for this sort of thing: https://github.com/localgovdrupal/localgov_services/issues/40